english/blah-balh/blah-blah.md

```diff
@@ -4,4 +4,6 @@ Be as smart as you can be in your voice so you don't sound stupid.
 
 And sing if possible. Singing cheers people up.
 
-If no sing [go back](../marshmallow.md)
\ No newline at end of file
+If no sing [go back](../marshmallow.md)
+
+Or go [drink a coffee](https://github.com/udacity/create-your-own-adventure/tree/master/english/coffee) to get your courage up
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.